### ISSUE TITLE
investigate ceph health warning

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1134]
```
[root@kzn-mon1 ~]# ceph health detail
HEALTH_WARN Reduced data availability: 1 pg inactive, 1 pg peering
PG_AVAILABILITY Reduced data availability: 1 pg inactive, 1 pg peering
    pg 133.1e is stuck peering for 49876.775876, current state peering, last acting [178,182,179,224,231,229,267,265,260]
  ```
  
The pg is stuck peering.

**Comment by naved001** [comment]
ceph cluster went into error state and now there are is some issues with the monitors :/

**Comment by naved001** [comment]
A short summary of things:

The peering pg was writing too many logs, which caused `/var/ceph/ceph.log` to get too big filling up `/var/` that can cause the monitors to fail (zabbix had an alert about low disk space, this [redhat document](https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/4/html-single/troubleshooting_guide/index#ceph-monitor-is-out-of-quorum_diag) also talks about how low space can cause issues). So, I cleared some space by removing some logs and powered off the host kzn-osd07 because it had a couple of drives in the warning state so I figured that it might be the one causing issues. I started the ceph-mon processes after doing a systemctl reset-failed, and then I could run the ceph command line. Now the cluster is healthy and the pg is no longer stuck peering.
